USAGE.md

```diff
@@ -0,0 +1,48 @@
+# config-parse(1) -- screwdriver config parser
```

Comment:
@d2lam I think it's a convention for man page formatting
http://liw.fi/manpages/